### DISCUSSION TITLE
having the default language routes without a prefix in app directory

### DISCUSSION THREAD

**Discussion by serge-ixor** [amannn/next-intl#182] [Q&A]
Hi,
I've been trying out the latest 2.11.0-beta.6 version with the app directory of Nextjs 13.
-> I've noticed that it currently does a redirect for the default language when no locale prefix is used. Is there an option to change this behaviour?
Example: is it possible to have **/about** be the default english about page without redirect to **/en/about**, and **/de/about** the german one, etc?

**Comment by amannn** [comment]
Hey! Yep, this is currently on my todo list: https://github.com/amannn/next-intl/pull/149. I'd like to support this out-of-the-box with the middleware.

**Reply by jasheal** [reply]
Would love this too!
